src/modules/acbs/acbs.service.ts

```diff
@@ -0,0 +1,41 @@
+import { HttpService } from '@nestjs/axios';
+import { Inject, Injectable } from '@nestjs/common';
+import { ConfigType } from '@nestjs/config';
+import AcbsConfig from '@ukef/config/acbs.config';
+import { AxiosError } from 'axios';
+import { catchError, lastValueFrom } from 'rxjs';
+
+import { AcbsPartyExternalRatingsResponseDto } from './dto/acbs-party-external-ratings-response.dto';
+import { AcbsException } from './exception/acbs.exception';
+import { AcbsResourceNotFoundException } from './exception/acbs-resource-not-found.exception';
+
+@Injectable()
+export class AcbsService {
+  constructor(
+    @Inject(AcbsConfig.KEY)
```

Comment:
do we need this here? in mdm we tend to use something like
```js
configService.get<string>('doc.description')
```
and we only need the configService from line 16

Reply:
We could do it that way instead if we prefer that style as team. Personally I prefer to inject the config because:
- if we inject the `configService`, then there's no type-safety around which config variables we'll use: the code can try to `get` any config key it wants. By injecting `config: ConfigType<typeof AcbsConfig>` instead, we get type-safety that we're only going to use config values that actually exist (and then we can use `Pick<ConfigType<typeof AcbsConfig>, 'baseUrl'>` to be a bit more specific that we only actually want `acbs.baseUrl`)
- it's easier for unit tests (and in general any code that wants to construct an `AcbsService`) to provide config options directly in the constructor than it is for them to provide a mock config service that can `get` the config options that the tests want to use

Reply:
ok cool. let's leave it as it is